src/ColumnSortable/Sortable.php

```diff
@@ -26,7 +26,13 @@ public function scopeSortable($query, $defaultSortParameters = null)
     {
         if (Request::filled('sort') && Request::filled('order')) {
             return $this->queryOrderBuilder($query, Request::only(['sort', 'order']));
-        } elseif ( ! is_null($defaultSortParameters)) {
+        }
+        
+        if (null === $defaultSortParameters) {
+            $defaultSortParameters = static::getDefaultSortable();
+        }
+        
+        if ( ! is_null($defaultSortParameters)) {
```

Comment:
formatting issue, it's better to be
`if (!is_null($defaultSortParameters)) {`
just for consistency